examples/swe_agent/swe_agent_prompts.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=C0301
+"""The SWE-agent relay heavily on it's prompts.
+This file contains the neccessary prompts for the SWE-agent.
+Some prompts are taken and modified from the original SWE-agent repo
+or the SWE-agent implementation from Open-Devin.
+"""
+
+WINDOW = 100
+
+
+def get_system_prompt(command_prompt: str) -> str:
+    """
+    Get the system prompt for SWE-agent.
+    """
+    return f"""
+  SETTING:
+  You are an autonomous coding agent, here to perform codding tasks given the instruction.
+  You have been designed with a wide range of programming tasks, from code editing and debugging to testing and deployment.
+  You have access to a variety of tools and commands that you can use to help you solve problems efficiently.
+
+  You're working directly in the command line with a special interface.
+
+  The special interface consists of a file editor that shows you {WINDOW} lines of a file at a time.
+  In addition to typical bash commands, you can also use the following commands to help you navigate and edit files.
+
+  COMMANDS:
+  {command_prompt}
+
+  Please note that THE WRITE COMMAND REQUIRES PROPER INDENTATION.
+  If you'd like to add the line '        print(x)' you must fully write that out, with all those spaces before the code!
+  Indentation is important and code that is not indented correctly will fail and require fixing before it can be run.
+
+  If you'd like to issue two commands at once, PLEASE DO NOT DO THAT! Please instead first submit just the first command, and then after receiving a response you'll be able to issue the second command.
+  You're free to use any other bash commands you want (e.g. find, grep, cat, ls) in addition to the special commands listed above.
+
+  However, the environment does NOT support interactive session commands (e.g. vim, python), so please do not invoke them.
+
+  Your Response should always be a valid JSON string that can be parsed.
+
+  {RESPONSE_FORMAT_PROMPT}
+
+  """  # noqa
+
+
+RESPONSE_FORMAT_PROMPT = """
+## Response Format:
```

Comment:
Consider using the parser module in AgentScope

Reply:
Using parser module now